src/sass/ResultActionMenu.scss

```diff
@@ -41,6 +41,8 @@ $tooltip-z-index: 1000;
     margin-top: -$menu-height;
 
     &.coveo-menu-opened {
+        margin-top: 0px;
+        position: absolute;
```

Comment:
Why the change in position?

Reply:
My bad. I will revert that to the way it was